app/boards/intel_adsp_cavs25_tgph.conf

```diff
@@ -29,6 +29,7 @@ CONFIG_COMP_VOLUME_WINDOWS_FADE=y
 CONFIG_COMP_DRC=y
 CONFIG_COMP_CROSSOVER=y
 CONFIG_COMP_MULTIBAND_DRC=y
+CONFIG_COMP_ARIA=y
```

Comment:
question still same: why choose tgl to enable ARIA?

Reply:
Why not? ARIA is already set to "y" for MTL and after. TGL is the best available generic developer platform for us, the UPX board. We have only high usage remote MTL platforms with usually no proper audio test env for them like I have on my own desk.

Reply:
We need to include new modules today so we can introduce on generic Linux for SOF v2.8